swift_browser_ui_frontend/package-lock.json

```diff
@@ -0,0 +1,11669 @@
+{
+  "name": "swift_browser_ui_frontend_npm",
+  "version": "0.3.3",
```

Comment:
updating version here as well?

Reply:
Oops, forgot to `npm up` when pushing new version, fixed now